source/first-steps/install/getting-meshroom/platforms/linux.rst

```diff
@@ -5,9 +5,9 @@ Linux
 
 .. code-block:: bash
 
-  tar -xf Meshroom-2020.1.0-linux-cuda10.tar.gz
+  tar -xf Meshroom-2021.1.0-linux-cuda10.tar.gz
```

Comment:
maybe we could just generalize so that we don't have to keep it up to date at each new release, something like
```suggestion
  tar -xf Meshroom-20XX.Y.Z-linux-cudaCC.tar.gz
```

Reply:
I did the changes for the two lines in the next commit !